apps/web/src/styles.css

```
@@ -281,3 +281,113 @@ html[data-hide-search-input] .search-input-wrapper {
+    opacity: 1;
+    filter: blur(0);
+  }
+  to {
+    transform: translate3d(-12%, 0, 0) scale(0.96);
+    opacity: 0;
+    filter: blur(4px);
+  }
+}
+
+@keyframes route-rise-from-right {
+  from {
```

Stylelint (17.9.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)